### PR TITLE
Docs revision #1051

### PR DESCRIPTION
**Pull request by oraNod**
https://github.com/infinispan/infinispan-operator/issues/1051

### CODE REVIEW

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+However, you must ensure that {brandname} clusters establish connections in the same way at each site.
 
 .Prerequisites
 
+Manually connecting {brandname} clusters to form cross-site views requires predictable network locations for {brandname} services.
+
+You need to know the network locations before they are created, which requires you to:
+
 * Have the host names and ports for each {brandname} cluster that you plan to configure as a backup location.
+* Have connection details for the `xsite` service for any remote {brandname} cluster that is running on {k8s}. +
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_configuring_sites_manually.adoc:18]: Probably, not sure about context
```suggestion
* Have connection details for the ` infinispan+xsite` service for any remote {brandname} cluster that is running on {k8s}. +
```

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+You must use the `xsite` service to form a cross-site view between a cluster that {ispn_operator} manages and any other cluster.
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_configuring_sites_manually.adoc:19]: The same as above
```suggestion
You must use the ` infinispan+xsite` service to form a cross-site view between a cluster that {ispn_operator} manages and any other cluster.
```

File: documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml [hunk 3]
```diff
@@ -14,7 +14,5 @@ spec:
         expose:
           type: ClusterIP
       locations:
-        - name: SiteA
-          url: infinispan+xsite://example-clustera-site
         - name: SiteB
-          url: infinispan+xsite://example-clusterb-site
+          url: infinispan+xsite://example-clusterb-site.<namespace>.svc:7900
```
Comment (dmvolod) [inline documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml:18]: 7900 is default value and can be omitted

Comment (oraNod) [reply documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml:18]: @Crumby Do you think we need the default port here?

**Review by rigazilla — APPROVED** [review]

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc (hunk 1 above)
Comment (dmvolod) [reply documentation/asciidoc/topics/proc_configuring_sites_manually.adoc]: Ops, I predict that I'm wrong about context :smile:

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc [hunk 4]
```diff
@@ -1,12 +1,22 @@
...
 
 .Prerequisites
 
+Manually connecting {brandname} clusters to form cross-site views requires predictable network locations for {brandname} services.
+
+You need to know the network locations before they are created, which requires you to:
+
 * Have the host names and ports for each {brandname} cluster that you plan to configure as a backup location.
+* Have connection details for the `infinispan+xsite` service for any remote {brandname} cluster that is running on {k8s}. +
+You must use the `infinispan+xsite` service to form a cross-site view between a cluster that {ispn_operator} manages and any other cluster.
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_configuring_sites_manually.adoc:19]: @oraNod please revert my commits, @Crumby points about context that I'm incorrectly define it

Comment (oraNod) [reply documentation/asciidoc/topics/proc_configuring_sites_manually.adoc]: @dmvolod yep, I noticed that and have reverted. all squashed now. please check the changes as I tried to clarify a bit more based on the points from @Crumby

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc [hunk 5]
```diff
@@ -1,12 +1,22 @@
...
+However, you must ensure that {brandname} clusters establish connections in the same way at each site.
 
 .Prerequisites
 
+Manually connecting {brandname} clusters to form cross-site views requires predictable network locations for {brandname} services.
+
+You need to know the network locations before they are created, which requires you to:
+
 * Have the host names and ports for each {brandname} cluster that you plan to configure as a backup location.
+* Have the host name of the `<cluster-name>-xsite` service for any remote {brandname} cluster that is running on {k8s}. +
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_configuring_sites_manually.adoc:18]: Oh, now I see the contexts `SiteServiceTemplate = "%v-site"`

```suggestion
* Have the host name of the `<cluster-name>-site` service for any remote {brandname} cluster that is running on {k8s}. +
```

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc [hunk 6]
```diff
@@ -1,12 +1,22 @@
...
+You must use the `<cluster-name>-xsite` service to form a cross-site view between a cluster that {ispn_operator} manages and any other cluster.
```
Comment (dmvolod) [inline documentation/asciidoc/topics/proc_configuring_sites_manually.adoc:19]:
```suggestion
You must use the `<cluster-name>-site` service to form a cross-site view between a cluster that {ispn_operator} manages and any other cluster.
```

File: documentation/asciidoc/topics/proc_configuring_sites_manually.adoc (hunk 5 above)
Comment (oraNod) [reply documentation/asciidoc/topics/proc_configuring_sites_manually.adoc]: @dmvolod thank you

File: documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml (hunk 3 above)
Comment (Crumby) [reply documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml:18]: Theoretically it should work without the port but I haven't tried it

Comment (oraNod) [reply documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml:18]: @dmvolod even if it's slightly unnecessary do you have any objection to including the default value? If @Crumby has tested it with the port then I'd like to include it.

**Review by pruivo — APPROVED** [review]

File: documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml (hunk 3 above)
Comment (dmvolod) [reply documentation/asciidoc/topics/yaml/xsite_cluster_ip.yaml:18]: Yes, sure, @oraNod let's keep it as is

**Review by dmvolod — APPROVED** [review]

**Pull request merged**